awscli/examples/mediaconvert/create-job-template.rst

```diff
@@ -0,0 +1,14 @@
+**To create a job template**
+
+The following example creates a job template with the transcoding settings that are specified in the file job-template.json that resides on your system:
+
+Command::
+     aws --endpoint-url=https://abcd1234.mediaconvert.region-name-1.amazonaws.com --region=region-name-1 mediaconvert create-job-template --name=JobTemplate1 --cli-input-json=file://~/job-template.json
+
+If you create your job template JSON file by using get-job-template and then modifying the file, remove the JobTemplate object, but keep the Settings child object inside it. Also, make sure to remove the following key-value pairs: LastUpdated, Arn, Type, and CreatedAt. You can specific the category, description, name, and queue either in the JSON file or at the command line.
+
+To get your account-specific endpoint, use describe-endpoints, or send the command without the endpoint. The service will return an error and your endpoint.
```

Comment:
Might be more helpful to show this
```
$ aws mediaconvert describe-endpoints --query "Endpoints[0].Url" --output text
https://abc123.mediaconvert.us-west-2.amazonaws.com
```

Reply:
> Looks like a good start.
> 
> This needs to be rebased off of develop so that there are not hundreds of commits.

Thanks for taking the time to review, @stealthycoin! Can you explain to me how to do that?

I get how to rebase to do things like squashing commits, but I don't understand anything at all about working from a fork. 

When I put up my PR, I tried to figure out how to make a fresh fork so that there wouldn't be a giant pile of commits, but it doesn't look like github will let me do that?

Reply:
> Might be more helpful to show this
> 
> ```
> $ aws mediaconvert describe-endpoints --query "Endpoints[0].Url" --output text
> https://abc123.mediaconvert.us-west-2.amazonaws.com
> ```
Can you explain why that's more helpful? It looks like you're setting up variable text for the endpoint and then feeding in that text maybe. I don't understand why that's simpler than directly specifying the endpoint, particularly for our customers, who will likely not have encountered having to use a custom endpoint before?